frontend/public/components/hpa.tsx

```diff
@@ -174,7 +174,7 @@ export const HorizontalPodAutoscalersDetails: React.SFC<HorizontalPodAutoscalers
     <MetricsTable obj={hpa} />
   </div>
   <div className="co-m-pane__body">
-    <Conditions obj={hpa} />
+    <Conditions obj={_.get(hpa, 'status.conditions')} />
```

Comment:
Looks like the `status.conditions` is always present on the HPA, in that case we dont need `_.get`